extensions/amp-a4a/0.1/signature-verifier.js

```diff
@@ -315,7 +315,8 @@ export class SignatureVerifier {
                   'Fast Fetch keyset spec requires Content-Type: ' +
                       'application/jwk-set+json');
               return response.json().then(
```

Comment:
Can we override the definition for `Response.p.json` to return a `!JsonObject`?

Reply:
I have no idea no to do this.
can you explain a bit?

Reply:
I don't either. I just don't want to always `/** @type {!JsonObject} */` when we call `Reponse.p.json`.

Reply:
How does dev.assertJson sound?